### ISSUE TITLE
Gadgetzan Bruiser not doing their work properly 

### ISSUE THREAD

**Issue by kalleas1** [Wall-core/Everlook-Bugtracker#659]
The guards attack me instead of the gankers, see vid.

The vid: https://youtu.be/NeE63SPadrY

**Comment by g-valentino** [comment]
Yep, Diabloxxx doing this sit with the pet in Booty Bay as well.

**Comment by Godmatik** [comment]
Bump - any update on this?
It also automatically puts you to 'at war' with Booty Bay so you have to turn it off.

**Comment by nicole-woebse** [comment]
I reported that in the Beta and it is still a thing. the guards attack you, even when you do nothing, no spells, no buffs on yourself, just riding you're darn mount while being attacked.

**Comment by Godmatik** [comment]
Per Wall I guess it's getting fixed on Wednesday with the next update
